assets/js/functions.js

```diff
@@ -18,3 +18,46 @@ function loadFaqComponent() {
         }
     );
 }
+
```

Comment:
```suggestion
```

Reply:
Can you do the above requested changes @gajindubandara